api/namespace/fund/data_store/store.py

```diff
@@ -66,6 +84,32 @@ def get_applications_for_fund(
         except KeyError:
             return f"Fund: {fund_name} not found.", 400
 
+    def search_applications(self, params):
```

Comment:
Lets see some comments on this function :) I am not as cleaver as you 💭

Reply:
Sorry will add